File: articles.jdbc/src/main/java/pet/db/jdbc/model/dto/Article.java
package pet.db.jdbc.model.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class Article {

    private Integer id;

    private LocalDateTime dateOfCreation;

    private String topic;

    private String content;

}

File: src/main/kotlin/pet/articles/model/dto/Article.kt
package pet.articles.model.dto

import kotlinx.serialization.Serializable
import pet.articles.tool.serializer.LocalDateTimeSerializer
import java.time.LocalDateTime

@Serializable
data class Article(
    val id : Int?,
    @Serializable(with = LocalDateTimeSerializer::class)
    val dateOfCreation : LocalDateTime,
    val topic : String,
    val content : String
)

